matlab/src/matlab/+arrow/+args/parseValidElements.m

```diff
@@ -0,0 +1,28 @@
+function validElements = parseValidElements(data, detectNulls, nullDetectionFcn)
```

Comment:
It seems that `nullDetectionFcn` isn't used.

Reply:
That's a mistake on my part. I meant to delete it.